_docs/README_ja.md

```diff
@@ -33,7 +33,7 @@ Usage: spoofdpi [options...]
         bypass DPI only on packets matching this regex pattern; can be given multiple times
   -port int
         port (default 8080)
-  -quiet
+  -silent
         do not write anything to standart output
```

Comment:
typo. standart -> standard
`do not show the banner and server information at start up` woud be better.

Reply:
Done